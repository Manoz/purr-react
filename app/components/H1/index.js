import styled from 'styled-components';

const H1 = styled.h1`
  margin-top: 1.2em;
  font-weight: 600;
  font-size: 40px;
  line-height: 1.09375;
  letter-spacing: .011em;
`;

export default H1;
