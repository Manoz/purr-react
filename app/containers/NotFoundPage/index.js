import React from 'react';
import { Link } from 'react-router-dom';

import H1 from 'components/H1';

const HomePage = () => (
  <section className="page home">
    <H1>404 page not found</H1>
    <p>
      <Link to='/'>Back to home</Link>
    </p>
  </section>
);

export default HomePage;
