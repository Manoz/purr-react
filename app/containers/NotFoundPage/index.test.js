import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import H1 from 'components/H1';
import NotFoundPage from './index';

describe('<NotFoundPage />', () => {
  it('should render a <section> tag', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper.find('section')).toHaveLength(1);
  });

  it('should have a <h1> title', () => {
    const wrapper = renderer.create(<H1 />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
