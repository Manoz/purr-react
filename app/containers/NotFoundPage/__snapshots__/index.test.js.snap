// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should have a <h1> title 1`] = `
.c0 {
  margin-top: 1.2em;
  font-weight: 600;
  font-size: 40px;
  line-height: 1.09375;
  -webkit-letter-spacing: .011em;
  -moz-letter-spacing: .011em;
  -ms-letter-spacing: .011em;
  letter-spacing: .011em;
}

<h1
  className="c0"
/>
`;
